aiconfig-docs/docs/gradio-workbook.md

```diff
@@ -0,0 +1,144 @@
+---
+sidebar_position: 21
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/tabConstants';
+
+# Gradio Workbooks
+
+**[Gradio Workbooks](https://huggingface.co/spaces/lastmileai/gradio-workbook-template) provide an out-of-the-box playground for creating Hugging Face Spaces.**
+
+The playground provides instant model selection, chaining of models, and templates - no setup or code required!
+
+<div align="center">
+  <picture>
+    <img alt="workbook" src="https://github.com/lastmile-ai/aiconfig/assets/81494782/494523e1-23c6-4709-94cb-7ebe86873dfd" width="600"/>
+  </picture>
+</div>
+<br/>\
+Gradio Workbook unlocks new capabilities to HuggingFace Spaces users!
```

Comment:
I think we should delete this line, we already say this basically in two sentences above the screenshot